### ISSUE TITLE
Actions operators

### ISSUE THREAD

**Issue by marcosh** [quickstrom/quickstrom#128]
Is there a way to combine two actions into one?

Something along the lines of

```
andThen :: Action -> Action -> Action
```

which would allow to define a single action

```
let ~focusAndClick = focus!(`foo`) andThen click!(`foo`);
```

More generally, is there a place where I can see in one place all the operators which are already defined and their type?

**Comment by owickstrom** [comment]
For your first question: no, actions are defined singularly. You can sort of emulate an action chain if you can use preconditions (e.g. ```focus(`foo`) when myPrecondition```), but it's perhaps a bit clumsy. Could it work for your case?

**Comment by marcosh** [comment]
> Second question: we have a basic docs generator, but sadly its output isn't published anywhere. That would be one way of getting the information you need.

Interesting! How do I run the docs generator? I tried using `pyright` but I couldn't find the right option

**Comment by owickstrom** [comment]
It's called `docstrom` and it's a part of Specstrom, the checker component used by Quickstrom. You can run `docstrom` for the Nix shell inside the Quickstrom repo, but it's really low-level and annoying. Ideally, it would be exposed through the Quickstrom CLI with all the include paths set correctly.

**Comment by marcosh** [comment]
Thanks, I tried it but I always get a `Command line:0:0: module not found` error message back. Probably I'm just using it wrong

**Comment by owickstrom** [comment]
Yes, that's probably because of missing include paths. I'll see if I can find some time to integrate it properly.
